recipe/build_openmm.bat

```diff
@@ -5,6 +5,11 @@ cd build
 
 set "CUDA_TOOLKIT_ROOT_DIR=%CUDA_PATH:\=/%"
 
+:: define NOMINMAX since openmm headers expect min/max to be functions not macros
+set "CFLAGS=%CFLAGS% -DNOMINMAX"
+set "CXXFLAGS=%CXXFLAGS% -DNOMINMAX"
```

Comment:
We should be able to remove these lines now.  That turned out not to be the problem.

Reply:
Done!